demos/src/demo-color.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const d3Selection = require('d3-selection');
+const colors = require('./../../src/charts/helpers/color');
+
+function createColors() {
+    const { colorSchemas } = colors;
+
+    const colorPalettes = Object.keys(colorSchemas).map(colorPalette => {
```

Comment:
That's awesome that you use functional approach to iterate over both color palettes and colorSchemas! I would suggest to use `forEach` instead since we use `.map` usually to return a new array with the same length. Since we are not using the `colorPalletes` array anywhere (you inject elements into the DOM), I would suggest to use `forEach` in both cases instead. Makes it more clear as to how it's being used. Nothing will change, `map` returns something and `forEach` doesn't. That's the only difference.

Reply:
Good point! Will update